### PR TITLE
fix: cluster tests stability

### PR DESCRIPTION
**Pull request by BorysTheDev**
make ports dynamic for cluster tests
close task for Proxy

### CODE REVIEW

File: tests/dragonfly/cluster_test.py [hunk 1]
```diff
@@ -1433,7 +1433,6 @@ async def test_migration_with_key_ttl(df_factory):
     assert await nodes[1].client.execute_command("stick k_sticky") == 0
 
 
-@pytest.mark.skip("test is flaky")
```
Comment (chakaz) [inline tests/dragonfly/cluster_test.py:1436]: Is this change intentional?
(I assume that changing the ports does not fix the test?)

**Review by adiholden — DISMISSED** [review]

**Review by chakaz — COMMENTED** [review]
While here, can you also change the hard-coded default port in line 924?

File: tests/dragonfly/cluster_test.py (hunk 1 above)
Comment (BorysTheDev) [reply tests/dragonfly/cluster_test.py:1436]: Thx

**Review by chakaz — APPROVED** [review]
